SketchUp/Sketchup/ComponentInstance.rb

```diff
@@ -488,10 +488,10 @@ def subtract(instance)
   #   new_transformation = Geom::Transformation.new([100,0,0])
   #   componentinstance.transform! new_transformation
   #
-  # @param transform
-  #   The transform object to apply to the component instance.
+  # @param transform [Geom::Transformation]
+  #   The transformation object to apply to the component instance.
   #
-  # @return status - true if successful, false if unsuccessful
+  # @return [Boolean] +true+ if successful, +false+ if unsuccessful
```

Comment:
For most boolean type of return I think ` @return [Boolean]` is enough - the "true is successful is redundant". We only need to add as much info as needed.